### PR TITLE
Rebuild app online

### PR DESCRIPTION
**Pull request by alexvandesande**

### CODE REVIEW

File: app/client/templates/views/send.html [hunk 1]
```diff
@@ -1,7 +1,5 @@
 <template name="views_send">
-
-    {{reactiveData}}
-
```
Comment (frozeman) [inline app/client/templates/views/send.html]: Why is this removed??
This was necessary to trigger reactivity in the respective function

File: app/client/templates/views/send.js [hunk 2]
```diff
@@ -101,38 +101,49 @@ Template['views_send'].onCreated(function(){
     TemplateVar.set('amount', '0');
     TemplateVar.set('estimatedGas', 300000);
     TemplateVar.set('sendAll', false);
+    
+    // Deploy contract
+    if(FlowRouter.getRouteName() === 'deployContract') {
+        TemplateVar.set('selectedAction', 'deploy-contract');
```
Comment (frozeman) [inline app/client/templates/views/send.js]: Only setting this here. Doesn't make itvreactive for token changes.

Comment (evertonfraga) [inline app/client/templates/views/send.js]: This is the PR link: https://github.com/ethereum/meteor-dapp-wallet/pull/361
@luclu would you elaborate how that works?
